src/main/java/com/orange/lo/sample/sqs/Mqtt2SqsApplication.java

```diff
@@ -1,20 +1,24 @@
-/** 
-* Copyright (c) Orange. All Rights Reserved.
-* 
-* This source code is licensed under the MIT license found in the 
-* LICENSE file in the root directory of this source tree. 
-*/
+/**
+ * Copyright (c) Orange. All Rights Reserved.
+ * <p>
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
 
 package com.orange.lo.sample.sqs;
 
+import com.orange.lo.sample.sqs.liveobjects.LoProperties;
+import com.orange.lo.sample.sqs.sqs.SqsProperties;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 
 @SpringBootApplication
+@EnableConfigurationProperties({LoProperties.class, SqsProperties.class})
```

Comment:
Why not use @Component on LoProperties class. If in the future you add new properties you will need to update this file instead of just create property bean using @Component

Reply:
Unfortunately it's not possible to use @ConstructorBinding and @Component together. It's a trade-off to have immutable configuration.